ydb/mvp/meta/meta.cpp

```diff
@@ -141,6 +141,15 @@ void TMVP::InitMeta() {
     MetaLocation.Endpoints.emplace_back("api", MetaApiEndpoint);
     MetaLocation.Endpoints.emplace_back("cluster-api", MetaApiEndpoint);
     MetaLocation.RootDomain = MetaDatabase;
+    MetaLocation.MetaDatabaseTokenName = MetaDatabaseTokenName;
+    switch (AccessServiceType) {
+        case NMVP::EAccessServiceType::YandexV2:
+            MetaLocation.MetaDatabaseTokenSource = TYdbLocation::EAuthTokenSource::Client;
```

Comment:
It is not propr. MetaLocation.MetaDatabaseTokenSource doesn't depend on AccessServiceType. We can make both variants of MetaDatabaseTokenSource for both variants of AccessServiceType. It must be a separate setting